lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -421,7 +466,98 @@
     acme_challenge_path: /etc/ssl/challenges/ansible.com/
     force: yes
 
+# The following example shows one assertonly usage using all existing options for
+# assertonly, and shows how to emulate the behavior with the openssl_certificate_info,
+# openssl_csr_info, openssl_privatekey_info and assert modules:
```

Comment:
...and then you have a user that wants to ensure `mode: u=rx`. ;-)

This is tough even with the "stats" module.

Reply:
Is it possible to use `assertonly` to check this? I'm not sure whether that actually works.

Also, you could use the `file` module with `check_mode: yes` to see whether there would be a change, and `assert` on that :)

Reply:
Hm, the `file` module in `check_mode` seems like the way to go then. Thanks for the tip! Maybe we should add it here too (e.g. `To check the additional parameters of this module also supported by the file module, you can use the file module in check_mode and see if the file is in the expected state.`)

Reply:
`assertonly` does not care about file permissions. I've tested it with `devel`, `stable-2.8`, `stable-2.7` and `stable-2.6`. If `mode` does not match the mode of the file in `path`, it doesn't complain or change anything.